tests/functional/raw-node/test/GCP/bucket/get.js

```diff
@@ -0,0 +1,181 @@
+const assert = require('assert');
+const async = require('async');
+const { GCP } = require('../../../../../../lib/data/external/GCP');
+const { makeGcpRequest } = require('../../../utils/makeRequest');
+const { gcpRequestRetry } = require('../../../utils/gcpUtils');
+const { getRealAwsConfig } =
+    require('../../../../aws-node-sdk/test/support/awsConfig');
+const constants = require('../../../../../../constants');
+
+const credentialOne = 'gcpbackend';
+
+describe('GCP: GET Bucket', function testSuite() {
+    this.timeout(180000);
+    let gcpClient;
+    let config;
+
+    before(() => {
```

Comment:
Seems like this `before` hook is unnecessary.

Reply:
Yeah, didn't catch that. Will remove the before hook.